src/System.Linq/tests/ZipTests.cs

```diff
@@ -11,6 +11,224 @@ namespace System.Linq.Tests
 {
     public class ZipTests : EnumerableTests
     {
+        [Fact]
```

Comment:
The tests are failing on non-.NET Core legs because the new surface area doesn't exist there, and thus the test won't compile.  Easiest way to address this is to separate these out into a ZipTests.netcoreapp.cs partial file, which is then conditionally-compiled, e.g. 
https://github.com/dotnet/corefx/blob/8305e5f139e83663b402673b71bb50e9fd0d2f31/src/System.Linq/tests/System.Linq.Tests.csproj#L6-L13